pkgs/by-name/zo/zotify/package.nix

```
@@ -0,0 +1,58 @@
+{
+  fetchFromGitHub,
+  python3Packages,
+  lib,
+}:
+python3Packages.buildPythonApplication rec {
+  pname = "zotify";
+  version = "0.6.13";
+
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "zotify-dev";
+    repo = "zotify";
+    # repository has no version tags
+    rev = "5da27d32a1f522e80a3129c61f939b1934a0824a";
+    hash = "sha256-KA+Q4sk+riaFTybRQ3aO5lgPg4ECZE6G+By+x2uP/VM=";
+  };
+
+  build-system = [ python3Packages.setuptools ];
+
+  nativeBuildInputs = [ python3Packages.pythonRelaxDepsHook ];
+
+  pythonRelaxDeps = [ "protobuf" ];
+
+  dependencies = with python3Packages; [
+    ffmpy
+    music-tag
+    pillow
+    tabulate
+    tqdm
+    librespot
+    pwinput
+    protobuf
+  ];
+
+  pythonImportsCheck = [
+    "zotify.album"
+    "zotify.app"
+    "zotify.config"
+    "zotify.const"
+    "zotify.loader"
+    "zotify.playlist"
+    "zotify.podcast"
+    "zotify.termoutput"
+    "zotify.track"
+    "zotify.utils"
+    "zotify.zotify"
+  ];
```

If the import of `zotify` fails then `zotify.album`, `zotify.app`, etc. will fail too. 

```suggestion
  pythonImportsCheck = [
    "zotify"
  ];
```

Checking the import of `zotify.album`, `zotify.app`, etc. would make sense if they depend on optional dependencies or have other requirements.